### ISSUE TITLE
png image not displayed properly

### ISSUE THREAD

**Issue by Dhara910** [littlevgl/lv_lib_lodepng#21]
i am trying to open .png file from SD card and display on LCD but it shows image with white background with nodata text.
how to resolve that?

**Comment by kisvegabor** [comment]
Probably the file can't be opened. Can you open and read a text file using [LVGL's file system](https://docs.lvgl.io/master/overview/file-system.html)?

**Comment by Dhara910** [comment]
yes, file opened and also it gives correct file size of png file

**Comment by kisvegabor** [comment]
Then probably it's an out of memory issue. Enabling [logging](https://docs.lvgl.io/master/porting/log.html) should help to see what is happening.
